gvsbuild/projects/librsvg.py

```diff
@@ -38,27 +38,23 @@ def __init__(self):
             ],
             patches=[],
         )
-        if Project.opts.enable_gi:
-            self.add_dependency("gobject-introspection")
-
-    def build(self):
-        self.builder.mod_env("INCLUDE", "include\\cairo", add_gtk=True)
-
-        b_dir = f"{self.builder.working_dir}\\{self.name}\\win32"
 
-        config = self.builder.opts.configuration
-        gtk_dir = self.builder.gtk_dir
-        rust_ver = Project.get_project("cargo").version
-        python = sys.executable
-        cmd = f'nmake -f makefile.vc CFG={config} "PREFIX={gtk_dir}" CARGO=cargo RUSTUP=rustup "PYTHON={python}" TOOLCHAIN_VERSION={rust_ver} install'
-
-        if Project.opts.enable_gi:
-            cmd += " INTROSPECTION=1"
+        if self.opts.enable_gi:
+            self.add_dependency("gobject-introspection")
+            enable_gi = "enabled"
+        else:
+            enable_gi = "disabled"
 
-        self.push_location(b_dir)
-        self.exec_vs(cmd)
-        self.pop_location()
+        self.add_param(f"-Dintrospection={enable_gi}")
+        self.add_param("-Ddocs=disabled")
+        self.add_param("-Dtests=false")
+        self.add_param("-Dvala=disabled")
 
+    def build(self):
+        self.builder.exec_cargo(
+            "install --locked --version 0.9.30+cargo-0.77.0 cargo-c"
```

Comment:
what are these versions?

Reply:
This is the version of cargo-c that is compatible with this older version of Cargo.